ChaosMod/Effects/db/Peds/PedsCatGuns.cpp

```diff
@@ -1,5 +1,8 @@
 #include <stdafx.h>
 
+static int MAX_CATS_COUNT = 20;
```

Comment:
Is it worth making this configurable? Even if that's a "hidden" config?

Reply:
what do you mean "configurable"?
defining it on top? letting the user change it? 🤔

Reply:
More or less? At least allow for more cats on more powerful machines? That can always be a later improvement though

Reply:
I don’t think it’s depending on „powerful machines“ but on the ped limit in game. As Long as we don’t have that implemented we should not be throwing objects around imho